README.md

```diff
@@ -27,7 +27,37 @@ For each **rollout group**, the operator **guarantees**:
 1. The maximum number of not-Ready pods in a StatefulSet doesn't exceed the value configured in the `rollout-max-unavailable` annotation (if not set, it defaults to `1`). Values:
   - `<= 0`: invalid (will default to `1` and log a warning)
   - `1`: pods are rolled out sequentially
-  - `> 1`: pods are rolled out parallelly (honoring the configured number of max unavailable pods)
+  - `> 1`: pods are rolled out in parallel (honoring the configured number of max unavailable pods)
+
+## How scaling up and down works
+
+The operator can also optionally coordinate scaling up and down of `StatefulSets` that are part of the same `rollout-group` based on the `grafana.com/rollout-downscale-leader` annotation. When using this feature, the `grafana.com/min-time-between-zones-downscale` label must also be set on each `StatefulSet`.
+
+This can be useful for automating the tedious scaling of stateful services like Mimir ingesters. Making use of this feature requires adding a few annotations and labels to configure how it works. Examples for a multi-AZ ingester group are given below.
+
+- For `ingester-zone-a`, add the following:
+  - Labels:
+    - `grafana.com/min-time-between-zones-downscale=12h` (change the value here to an appropriate duration)
```

Comment:
I'd like to make both of these labels into annotations because they don't _need_ to be labels but wanted to get this functionality rolled out first. We'll have to coordinate with Loki for that change.